how-to-use-azureml/machine-learning-pipelines/intro-to-pipelines/aml-pipelines-schedule-with-pipeline-parameter-built-in-functions.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to Setup a Schedule with Pipeline Parameter Expressions\n",
+    "In this notebook, we will show you how to create/update a schedule with supported pipeline parameter expressions, and expressions we support."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction of pipeline parameter expression"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- What is pipeline parameter expression  \n",
+    "  \n",
+    "    Pipeline parameter allows user dynamically running pipeline experiments with different inputs and parameters without any code change. Compared to manual modification of parameter values each run, pipeline parameter expression can work as a more advanced mode by getting their values in runtime, which can relieve pain points in both regular run scheduling and pipeline iterative development. \n",
+    "      "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- How to use pipeline parameter expression  \n",
+    "  \n",
+    "    See below table for supported built-in functions in schedule, a function invoked with an identifier \"@\" at the beginning of it will turn into a pipeline parameter expression. For pipeline parameter expression containing nested functions like \"@formatDateTime(subtractFromTime(utcNow(),1,'Day'),'yyyy-MM-dd')\", use only one \"@\" at the begining of the expression, there is no need to add identifier to each of the functions.\n",
```

Comment:
You could use invocation identifier `@` at the beginning of expression, and the expression value will be resolved in runtime. 
For pipeline parameter expression containing nested functions like "@formatDateTime(subtractFromTime(utcNow(),1,'Day'),'yyyy-MM-dd')", use only one "@" at the begining of the expression, there is no need to add identifier to each of the functions.
See below table for supported built-in functions in schedule.